libs/application/templates/public-debt-payment-plan/src/dataProviders/paymentPlanPrerequisites.ts

```diff
@@ -1,44 +1,224 @@
+import {
+  PaymentScheduleConditions,
+  PaymentScheduleDebts,
+  PaymentScheduleEmployer,
+  PaymentScheduleInitialSchedule,
+} from '@island.is/api/schema'
 import {
   BasicDataProvider,
-  SuccessfulDataProviderResult,
   FailedDataProviderResult,
+  SuccessfulDataProviderResult,
 } from '@island.is/application/core'
-import { Prerequisites } from './tempAPITypes'
-
-const createPrerequisitesResponse = (): Prerequisites => {
-  return {
-    maxDebt: 2000000,
-    totalDebt: 500000,
-    disposableIncome: 300000,
-    alimony: 0,
-    minimumPayment: 30000,
-    maxDebtOk: true,
-    maxDebtText: 'Max debt was not OK :(',
-    taxesOk: true,
-    taxesText: 'Taxes was not OK :(',
-    taxReturnsOk: true,
-    taxReturnsText: 'Tax returns was not OK :(',
-    vatOk: true,
-    vatText: 'VAT was not OK :(',
-    citOk: true,
-    citText: 'CIT was not OK :(',
-    accommodationTaxOk: true,
-    accommodationTaxText: 'Accommodation tax was not OK :(',
-    withholdingTaxOk: true,
-    withholdingTaxText: 'Witholding tax was not OK :(',
-    wageReturnsOk: true,
-    wageReturnsText: 'Wage returns was not OK :(',
+
+const queryPaymentScheduleDebts = `
+  query PaymentScheduleDebts {
+    paymentScheduleDebts {
+      nationalId
+      type
+      paymentSchedule
+      organization
+      explanation
+      totalAmount
+      chargetypes {
+        id
+        name
+        principal
+        intrest
+        expenses
+        total
+      }
+    }
+  }		
+`
+
+const queryPaymentScheduleConditions = `
+  query PaymentScheduleConditions {
+      paymentScheduleConditions {
+        nationalId
+        maxDebtAmount
+        maxDebtAmount
+        totalDebtAmount
+        minPayment
+        maxPayment
+        collectionActions
+        doNotOwe
+        maxDebt
+        oweTaxes
+        disposableIncome
+        taxReturns
+        vatReturns
+        citReturns
+        accommodationTaxReturns
+        withholdingTaxReturns
+        wageReturns
+        alimony
+    }
   }
+`
+
+const queryPaymentScheduleEmployer = `
+  query PaymentScheduleEmployer {
+    paymentScheduleEmployer {
+      nationalId    
+      name
+    }
+  }
+`
+
+const queryPaymentScheduleInitialSchedule = `
+	query PaymentScheduleInitialSchedule($input: GetInitialScheduleInput!) {
+		paymentScheduleInitialSchedule (input : $input){
```

Comment:
Whats this indent?

Reply:
It's a mystery! A mystery that will not be solved, but fixed!